examples/complete/versions.tf

```diff
@@ -9,7 +9,7 @@ terraform {
 
     snowflake = {
       source  = "Snowflake-Labs/snowflake"
-      version = "~> 0.57.0"
+      version = ">= 0.64.0"
```

Comment:
"~> 0.64.0" This might be better? It only upgrades to latest minor version and allows us to specify explicitly when you want to upgrade major versions.

Reply:
Makes sense 👍